### ISSUE TITLE
What's the function of "dp_grpc_server_addr" channel?

### ISSUE THREAD

**Issue by dingrui37** [p4lang/behavioral-model#414]
In my opnion, all packets can be sent over "grpc_server_addr" channel,  why we need "dp_grpc_server_addr" ?
![default](https://user-images.githubusercontent.com/22718365/28412100-b906dbac-6d75-11e7-8fc2-1cd194d8d71d.png)

I also confused by test_grpc_dp.cpp code, after complie the P4Runtiem.proto, there isn't a package called "p4::bm", so where does "p4::bm::DataplaneInterface" come from? 
```
dataplane_stub(p4::bm::DataplaneInterface::NewStub(
            dataplane_channel))
```

**Comment by antoninbas** [comment]
This second gRPC service is used to inject & receive data packets into the dataplane, not controller packets... It is meant for unit testing only and is not meant to be used by the controller. As indicated in the name, it is specific to the behavioral-model. It is for people who want to test a P4 program with `simple_switch_grpc` without using veth interfaces (for example), which require some special capabilities based on your OS, and prefer to inject and receive packets through a gRPC channel.
In a nutshell it is completely independent from P4Runtime.
